<launch>
  <group name="wg">
    <node pkg="mechanism_control" type="spawner.py" args="$(find wg_robot_description)/laser_tilt/laser_tilt_controller.xml" />
    <node pkg="pr2_mechanism_controllers" type="control_laser.py" args="laser_controller sine 1 .45 .40" />

    <node pkg="point_cloud_assembler" type="point_cloud_assembler" output="screen"  name="assembler">
      <remap from="scan" to="tilt_scan"/>
      <param name="ignore_laser_skew" type="bool" value="true" />
    </node>

    <node pkg="point_cloud_assembler" type="point_cloud_snapshotter" output="screen" name="snapshotter">
      <remap from="laser_scanner_signal" to="laser_controller/laser_scanner_signal"/>
      <param name="fixed_frame" type="string" value="torso_lift_link" />
    </node>

    <!--param name="ransac_ground_plane_extraction/listen_topic" value="full_cloud"/>
    <param name="ransac_ground_plane_extraction/publish_ground_plane_topic" value="ground_plane"/>
    <param name="ransac_ground_plane_extraction/publish_obstacle_topic" value="obstacle_cloud"/>

    <param name="ransac_ground_plane_extraction/min_ignore_distance" value="-0.01"/>
    <param name="ransac_ground_plane_extraction/max_ignore_distance" value="0.01"/>
    <param name="ransac_ground_plane_extraction/distance_threshold" value="0.01"/>
    <param name="ransac_ground_plane_extraction/filter_delta" value="0.5"/>
    <param name="ransac_ground_plane_extraction/max_ransac_iterations" value="500"/>

    <node pkg="ransac_ground_plane_extraction" type="ransac_ground_plane_node"/-->

  </group>
</launch>